reference/7.1/Microsoft.PowerShell.Utility/Get-Random.md

```diff
@@ -42,8 +40,20 @@ The `Get-Random` cmdlet gets a randomly selected number. If you submit a collect
 Without parameters or input, a `Get-Random` command returns a randomly selected 32-bit unsigned
 integer between 0 (zero) and **Int32.MaxValue** (`0x7FFFFFFF`, `2,147,483,647`).
 
-You can use the parameters of `Get-Random` to specify a seed number, minimum and maximum values, the
-number of objects returned from a submitted collection, and the entire collection in a random order.
+By default, `Get-Random` generates cryptographically secure randomness using the
+[RandomNumberGenerator](/dotnet/api/system.security.cryptography.randomnumbergenerator) class.
+
+You can use the parameters of `Get-Random` to specify a seed number, minimum and maximum values, and
```

Comment:
```suggestion
You can use the parameters of `Get-Random` to specify a seed number, minimum and maximum values for random selection, and
```

Reply:
I don't know if this is needed, it just tripped me up a little bit on the first read through.